src/foundations/page-template/example-basic.njk

```diff
@@ -21,4 +21,4 @@ layout: ~
     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
     <p>Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>
     <p>Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
-{% endblock %}
+{% endblock %}
```

Comment:
The line on the end of this file shouldnt really have been removed

Reply:
For some reason git is showing the line being removed and then added back in the same place. So looks like the line is still there

Reply:
Ah weird